src/express.js

```diff
@@ -1,15 +1,28 @@
+const os = require('os');
+const cluster = require('cluster');
+const http = require('http');
+const express = require('express');
+const debug = require('debug')('cnn-server:express');
+const compression = require('compression');
+const { healthcheckRoute } = require('./healthcheck.js');
+const { handleNoMatch, handleError } = require('./errors.js');
+
 function start(app, config) {
-    const { port } = config;
-    const http = require('http');
-    http.createServer(app).listen(port, () => {
-        log.important(`Service started on port: ${port}`);
-    });
-    // @TODO: Add error check
-    // @TODO: Add support for https
+    if (process.env.ENABLE_CLUSTER && cluster.isMaster) {
+        os.cpus().forEach(c => cluster.fork());
+        cluster.on('exit', function fork(worker) {
+            log.fatal(`Worker ${worker.id} died.`);
```

Comment:
It would be nice if inform that the worker die but we are restarting it. Something like:  log.fatal(`Worker ${worker.id} died. Restating it.`);

Reply:
agreed. done.